components/console-backend-service/internal/domain/serverless/function_converter.go

```diff
@@ -88,6 +98,11 @@ func (c *functionConverter) ToFunction(name, namespace string, in gqlschema.Func
 	envVariables := c.fromGQLEnv(in.Env)
 	minReplicas, maxReplicas := c.fromGQLReplicas(in.Replicas)
 
+	runtime := v1alpha1.Runtime("")
```

Comment:
If the runtime is optional, is it needed to cast first to empty string, not nil?